Snippets/RawMessaging/Core_8/Usage.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Routing;
+using NServiceBus.Transport;
+
+public class Usage
+{
+    static async Task Start()
+    {
+        #region Configuration
+
+        var transport = new LearningTransport();
+
+        var hostSettings = new HostSettings(
+            name: "MyRawEndpoint",
+            hostDisplayName: "My Raw Endpoint",
+            startupDiagnostic: new StartupDiagnosticEntries(),
+            criticalErrorAction: (message, exception, token) =>
+            {
+                Console.WriteLine("Critical error: " + exception);
+            },
+            setupInfrastructure: true);
+
+        var infrastructure = await transport.Initialize(hostSettings, new[]
+            {
+                new ReceiveSettings(
+                    id: "1",
```

Comment:
Maybe an ID that makes sense? Why a number?
```suggestion
                    id: "Primary",
```

Even looking at that I don't understand why it has an ID at all. What does it internally need that ID for?